### DISCUSSION TITLE
希望增加一个小小的功能

### DISCUSSION THREAD

**Discussion by BaiXianMoYun** [the1812/Bilibili-Evolved#1778] [综合讨论 / General]
在收藏夹自动播放视频时，能不能跳过一个多p视频的第几个p啊，有些视频两三个p都是一样的内容。

**Comment by Coulomb-G** [comment]
这你应该去找UP主,我们怎么知道哪几P是重复的内容?

**Reply by 1970633640** [reply]
可能是最后1p，防止撞车发一个无意义视频。不过这种up主我是瞧不起的，恶意规避防撞车检测。

**Reply by BaiXianMoYun** [reply]
> 这你应该去找UP主，我们怎么知道哪几P是重复的内容？

我的意思是，有些up会在之后追加另一个分辨率的视频，但内容本质与p1一样。

**Reply by the1812** [reply]
那你就点下一个视频啊

**Reply by BaiXianMoYun** [reply]
> 那你就点下一个视频啊

所以才希望多一个可以自动跳过这个么一个功能啊。

**Reply by the1812** [reply]
不太现实, 脚本可不知道这几个视频一不一样

**Reply by BaiXianMoYun** [reply]
> 不太现实, 脚本可不知道这几个视频一不一样

我不是说让脚本检测视频是否一样，自动跳过一个视频的p2+++++就够了

**Reply by the1812** [reply]
这种功能也太怪了吧, 我也建议你去找up主, 单独1p本来就能有多个分辨率, 为什么还要分p

**Reply by BaiXianMoYun** [reply]
> 这种功能也太怪了吧，我也建议你去找up主，单独1p本来可以有多个分辨率，为什么还要分p

这个我就不知道了，但大部分up的高分辨率都是过段时间追加的，那么多，一个个找感觉不太现实

**Reply by the1812** [reply]
那也是up的问题, 高分辨率追加也是不需要多分p的, 我只能提供点建议:
- 键盘快捷键<kbd>]</kbd>可以直接下一p
- 关闭自动连播可以阻止自动下一p
- 当然, 直接点下一个视频还是最省事的

**Reply by BaiXianMoYun** [reply]
> 那也是up的问题, 高分辨率追加也是不需要多分p的, 我只能提供点建议:
> 
> * 键盘快捷键]可以直接下一p
> * 关闭自动连播可以阻止自动下一p
> * 当然, 直接点下一个视频还是最省事的

果然是我放一大堆视频在一个收藏夹连播的这个需求很怪么，谢谢了

**Reply by the1812** [reply]
连播本来不是啥问题, 问题在于你说的这些up乱分p.
总之我是拒绝因为某些up的个人行为而去专门定制功能的, 或许你可以去 [GreasyFork论坛](https://greasyfork.org/zh-CN/discussions/requests) 问问有没有人愿意做.
